packages/wrangler/src/queues/client.ts

```diff
@@ -121,6 +150,22 @@ export interface ConsumerResponse extends PostConsumerBody {
 	dead_letter_queue?: string;
 }
 
+export async function deletePullConsumer(
+	config: Config,
+	queueName: string
+): Promise<void> {
+	const accountId = await requireAuth(config);
+	const queue = await getQueue(config, queueName);
+	const consumer = queue.consumers[0];
+	if (consumer?.type !== "http_pull") {
+		throw new UserError(`No http_pull consumer exists for queue ${queueName}`);
+	}
```

Comment:
`UserError` checking that this is not a worker consumer to prevent accidental removal of a different type of consumer

Reply:
Nice! Good use of `UserError` to prevent unwanted Sentry notifications for us.